metron-platform/metron-common/src/main/scripts/stellar

```diff
@@ -28,10 +28,15 @@ elif [ -e /usr/lib/bigtop-utils/bigtop-detect-javahome ]; then
   . /usr/lib/bigtop-utils/bigtop-detect-javahome
 fi
 
+export METRON_SYSCONFIG="/etc/default/metron"
+if [ -f "$METRON_SYSCONFIG" ]; then
+	source $METRON_SYSCONFIG
```

Comment:
In `/etc/default/metron`, `METRON_JVMFLAGS` will be set to either...
1. Empty if the cluster is not kerberized 
1. Set to `-Djava.security.auth.login.config=$METRON_HOME/client_jaas.conf` if kerberos has been enabled.